### PR TITLE
Bug: Access Review changes in expand view

### PR DESCRIPTION
**Pull request by a2batic**
Signed-off-by: Kanika <kmurarka@redhat.com>

### CODE REVIEW

File: frontend/packages/ceph-storage-plugin/src/plugin.ts [hunk 1]
```diff
@@ -178,6 +178,11 @@ const plugin: Plugin<ConsumedExtensions> = [
             throw e;
           });
       },
+      accessReview: {
+        group: models.OCSServiceModel.apiGroup,
+        resource: models.OCSServiceModel.plural,
+        verb: 'patch',
```
Comment (spadgett) [inline frontend/packages/ceph-storage-plugin/src/plugin.ts]: You need the resource name and namespace. You're checking if you can patch every OCS service in all namespaces.

Comment (a2batic) [reply frontend/packages/ceph-storage-plugin/src/plugin.ts]: ack

**Comment by a2batic** [comment]
@spadgett addressed review commnets, please review

File: frontend/packages/ceph-storage-plugin/src/plugin.ts [hunk 2]
```diff
@@ -178,6 +178,11 @@ const plugin: Plugin<ConsumedExtensions> = [
             throw e;
           });
       },
+      accessReview: {
```
Comment (spadgett) [inline frontend/packages/ceph-storage-plugin/src/plugin.ts]: This should be a function that takes in the obj and kind and returns the access review object.

File: frontend/packages/operator-lifecycle-manager/src/components/operand.tsx [hunk 3]
```diff
@@ -73,6 +73,11 @@ const getActions = (selectedObj: any) => {
   const pluginActions = actions.map((action) => (kind, ocsObj) => ({
     label: action.properties.label,
     callback: action.properties.callback(kind, ocsObj),
+    accessReview: {
+      ...action.properties.accessReview,
+      name: ocsObj.metadata.name,
+      namespace: ocsObj.metadata.namespace,
```
Comment (spadgett) [inline frontend/packages/operator-lifecycle-manager/src/components/operand.tsx]: I don't think we can make this assumption. The action will not always directly work on the same object, so we need to get the `accessReview` object as a callback

**Comment by a2batic** [comment]
PR is obsolete

**Pull request closed without merging**